.github/workflows/pr.yaml

```diff
@@ -20,6 +20,10 @@ jobs:
       - conda-python-other-tests
       - conda-java-tests
       - conda-notebook-tests
+      - cudf-wheel-build
```

Comment:
Wondering if we should align the first word of the job, somehow -- "conda" and "wheel". Maybe `wheel-build-cudf`, `wheel-tests-cudf`, `wheel-build-dask-cudf`, `wheel-tests-dask-cudf`?